bundle/src/main/java/com/adobe/acs/commons/remoteassets/impl/RemoteAssetsNodeSyncScheduler.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.remoteassets.impl;
+
+import com.adobe.acs.commons.remoteassets.RemoteAssetsNodeSync;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.Designate;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Scheduled Service that runs the Remote Assets node sync.
+ *
+ * This job will sync asset nodes based on OSGi configuration.
+ */
+@Component(
```

Comment:
For the life of me I cannot figure out why I can't get this scheduler to run.  I've added a config via felix console (to satisfy `configurationPolicy=REQUIRE` and used a scheduler expression of `0 * * ? * *` (every minute, just for testing purposes) but it never runs.  It seems every other scheduler in ACS Commons has slightly different Annotations as well, so I'm not sure which pattern to follow.

Anyone have any ideas on what I have incorrect here?

Reply:
Do I maybe need a reference to a `Scheduler` and then need to add/remove my object?  I dont see any of the other schedulers in ACS Commons doing that though...

Reply:
This is now resolved.